### ISSUE TITLE
You copy this repo from this https://github.com/darkslayer92/E-Commerce-Application-Nodejs-Angular-8-MYSQL

### ISSUE THREAD

**Issue by victorray84** [arvind5588/ecommerce-app#6]
Why did u copy from this site: https://github.com/darkslayer92/E-Commerce-Application-Nodejs-Angular-8-MYSQL

**Comment by arvind5588** [comment]
> Why did u copy from this site: https://github.com/darkslayer92/E-Commerce-Application-Nodejs-Angular-8-MYSQL

Yes, because I had already a reference copy of this before 6-7 month before, and I have also modified the API getProductDetails where middleware is implemented to check if a product's 10 qty is already sold if already sold it will through an error stating that "Product is out of stock".

**Comment by victorray84** [comment]
Thanks for pointing me this thing. I thought u copied all the things.
Please change the structure.
